contrib/docker/Dockerfile.alpine

```diff
@@ -1,12 +1,12 @@
-FROM alpine:3.14.3
+FROM alpine:3.16
 LABEL org.opencontainers.image.authors="Vincenzo Palazzo (@vincenzopalazzo) vincenzopalazzodev@gmail.com"
 
 WORKDIR /build
 
 RUN apk update && \
     apk add ca-certificates alpine-sdk autoconf automake git libtool \
     gmp-dev sqlite-dev python3 py3-mako net-tools zlib-dev libsodium gettext su-exec \
-    python3 py3-pip #&& \
+    ;
```

Comment:
It's probably sufficient to remove the `\` on line 8 and we can drop this line altogether.

Also notice that the next line is commented out, so we should remove that as well.